src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1533,8 +1533,8 @@ public override BoundNode VisitDelegateCreationExpression(BoundDelegateCreationE
 
             static bool ignoreReceiver(BoundExpression receiver, MethodSymbol method)
             {
-                // ignore the implicit `this` receiver on a static method
-                return method.IsStatic && receiver is { Kind: BoundKind.ThisReference, WasCompilerGenerated: true };
+                // static methods that aren't extensions get an implicit `this` receiver that should be ignored
```

Comment:
Do we have a test for an extension method called in non-reduced form? It may not be very meaningful since I think this would either have no receiver or have a type expression as a receiver, but thought I would check anyway.

Reply:
the static method itself doesn't get an implicit receiver - but the methodgroup...